patroni/api.py

```diff
@@ -89,8 +90,17 @@ def do_GET(self, write_status_code_only=False):
         patroni = self.server.patroni
         cluster = patroni.dcs.cluster
 
-        replica_status_code = 200 if not patroni.noloadbalance and \
-            response.get('role') == 'replica' and response.get('state') == 'running' else 503
+        replica_status_code = 503
+        if response.get('role') == 'replica' and response.get('state') == 'running' \
+           and not patroni.noloadbalance:
+            max_replica_lag = self.path_query.get('lag') and parse_int(self.path_query.get('lag')[0]) or 0
```

Comment:
The `parse_int()` function optionally accepts unit as a second argument. It allows doing very neat tricks like specifying lag not only in bytes, but in the arbitrary unit:
```python
In [2]: parse_int('16MB', 'B')
Out[2]: 16777216
```
If the value doesn't have the unit it would still work as previously:
```python
In [3]: parse_int('16777216', 'B')
Out[3]: 16777216
```

Reply:
Thanks for pointing it out. Will update.